backend/handlers/handlers.go

```diff
@@ -900,6 +900,11 @@ func (handlers *Handlers) getExchangeMoonpayBuy(r *http.Request) (interface{}, e
 	if acct == nil {
 		return nil, fmt.Errorf("unknown account code %q", acctCode)
 	}
+
+	if err := acct.Initialize(); err != nil {
```

Comment:
Oh, I get it now. Before, it was in account handlers which I guess was initialized. But now in the main handlers, it might not have been initialized yet. Does that sounds right?

Reply:
(by "it" I meant `getExchangeMoonpayBuy` handler method)

Reply:
it was also not initialized before either.

it was accidentally initialized by the portfolio (chart.go), but if you click fast enough, it will not have gotten to Litecoin yet ;)